deploy/addons/registry/registry-rc.yaml

```diff
@@ -9,11 +9,11 @@ metadata:
 spec:
   replicas: 1
   selector:
-    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/minikube-addons: registry
   template:
     metadata:
       labels:
-        addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Is it required for the pod manifest as well? I guess replication controllers don't work the same as deployments. I'll add this back.